hudi-aws/pom.xml

```diff
@@ -34,6 +34,12 @@
     </properties>
 
     <dependencies>
+        <!-- Logging -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-1.2-api</artifactId>
+        </dependency>
```

Comment:
Should this already be included as a transitive dependency, introduced by `hudi-common` module?

Reply:
"provided" deps are not included transitively

Reply:
Makes sense.  I noticed that later.